src/content/peerconnection/webaudio-input/js/main.js

```diff
@@ -41,8 +41,7 @@ function start() {
     video: false
   };
   navigator.mediaDevices.getUserMedia(constraints)
-  .then(gotStream)
-  .catch(gotStreamFailed);
+    .then(handleSuccess).catch(handleFailure);
```

Comment:
Also moved the dot back to the previous line for all these (so the line starts with `then` not `.then`, etc.) Not sure what's best — but it's probably good if we stick to one style.